.github/workflows/build_test_and_deploy.yml

```
@@ -0,0 +1,91 @@
+# Copyright (c) 2022 ETH Zurich
+#
+# SPDX-License-Identifier: BSL-1.0
+# Distributed under the Boost Software License, Version 1.0. (See accompanying
+# file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+name: Build, test and deploy docker images
+
+# The builds are triggered with pushes on the 'main' branch or with a pull
+# request against the 'main' branch
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  # Build base docker image
+  build_base_image:
+    env:
+      version: 1
+      image_name: pika-ci-base
+    # Cannot use the env variables in the name
+    name: github/pika-ci-base
```

```suggestion
    name: base
```
The `github/` prefix that is there on the main `pika` repo is just to namespace the workflow names for bors. We don't really need that here.

---

sorry copied everything ^^'